databuilder/models/table_metadata.py

```diff
@@ -150,16 +199,16 @@ class ColumnMetadata:
     COLUMN_DESCRIPTION = 'description'
     COLUMN_DESCRIPTION_FORMAT = '{db}://{cluster}.{schema}/{tbl}/{col}/{description_id}'
 
-    # Relation between column and tag
-    COL_TAG_RELATION_TYPE = 'TAGGED_BY'
-    TAG_COL_RELATION_TYPE = 'TAG'
+    # Relation between column and badge
+    COL_BADGE_RELATION_TYPE = 'HAS_BADGE'
```

Comment:
I see one issue: because `ColumnMetadata` is not a Neo4jCsvSerializable, I don't think we are able to publish column level badge without creating the table metadata model as column node is created part of table model...

In this case, I think we may still need to update the metadata put endpoint to support column badge. WDYT? @allisonsuarez @jinhyukchang